File: Problem002.py
from ProjectEuler import Fibonacci as F
#   This doesn't work
a = 2
max_value = 10
val = F(a)
values = [val]
while(a < max_value):
    a += 1
    val = F(a)
    values.append(val)



print(values)

File: Problem002.js
function Fibonacci(n) {
  if(n == 0) {
    return 0;
  }
  else if(n == 1) {
    return 1;
  }
  else {
    return Fibonacci(n - 1) + Fibonacci(n - 2);
  }
}

var a = 1;
var max_value = 1000;
var values = [];
while(Fibonnaci(a) < max_value) {
  if(Fibonnaci(a) % 2 == 0) {
    
  }
}
phantom.exit();
